src/libsync/vfs/cfapi/cfapiwrapper.cpp

```diff
@@ -40,6 +41,9 @@ Q_LOGGING_CATEGORY(lcCfApiWrapper, "nextcloud.sync.vfs.cfapi.wrapper", QtInfoMsg
       FIELD_SIZE( CF_OPERATION_PARAMETERS, field ) )
 
 namespace {
+constexpr auto syncRootFlagsFull = 34;
+constexpr auto syncRootFlagsNoCfApiContextMenu = 2;
```

Comment:
@mgallien These are not documented! I have these values experimentally. Also, check this article https://connect.resilio.com/hc/en-us/articles/360015676480-No-native-Selective-Sync-context-menu-items-on-Windows They confirm my findings but do not provide a link to the documentation.
And, no. There is no way to hide just "Make always available locally" and display "Free up local space". There is no valid flag combination for it. Could be this page is related https://docs.microsoft.com/en-us/windows/win32/shell/sfgao, but, I am not sure. Microsoft does not explain it.

Reply:
too bad